emission/core/wrapper/suggestion_sys.py

```diff
@@ -164,23 +173,34 @@ def is_service_nominatim(business):
     return True
 
 '''
-NOMINATIM VERS: Function that RETURNS a list of categories that the business falls into 
+NOMINATIM VERS: Function that RETURNS a list of categories that the business falls into
 
-Using the Google reverse lookup in the except clause, in case Nominatim's results are too vague. 
-Will first try Nominatim's reverse lookup, but if Nominatim returns a broad "address" 
-of the street and the city, without a full address with a specific location 
+Using the Google reverse lookup in the except clause, in case Nominatim's results are too vague.
+Will first try Nominatim's reverse lookup, but if Nominatim returns a broad "address"
+of the street and the city, without a full address with a specific location
 Such as Piedmont Ave, Berkeley, CA
 
-Then the function will enter the Google reverse lookup and choose a business that is closest to 
+Then the function will enter the Google reverse lookup and choose a business that is closest to
 latitude and longitude given
 '''
 
+
 ### BEGIN: Pulled out candidate functions so that we can evaluate individual accuracies
 def find_destination_business_google(lat, lon):
     return return_address_from_google_nomfile(lat, lon)
 
 def find_destination_business_yelp(lat, lon):
-    return (None, None, None, None)
+    yelp_from_lat_lon = newsearch(YELP_API_KEY, lat, lon, 50)
+    print(yelp_from_lat_lon)
+    if yelp_from_lat_lon == {}:
+        return None, None
```

Comment:
The `find_destination_business` step of the pipeline returns 4 values. Please make your candidate step consistent with the other standard.